procmon_parser/symbol_resolver/symbol_resolver.py

```diff
@@ -0,0 +1,711 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
+"""Module used to resolve symbolic information for a given a stack trace.
+"""
+import ctypes
+import enum
+import logging
+import os
+import platform
+import re
+import sys
+import winreg
+
+import procmon_parser
+
+if sys.platform != "win32":
+    raise RuntimeError("Symbol Resolver can only be used on Windows Operating Systems.")
+
+if sys.version_info >= (3, 5, 0):
+    import typing
+    import pathlib  # TODO: to be converted to py 2.7 equivalent.
+
+from procmon_parser.symbol_resolver.win.dbghelp import (
+    DbgHelp, PFINDFILEINPATHCALLBACK, SYMBOL_INFOW, IMAGEHLP_LINEW64, SYMOPT, SSRVOPT, PSYMBOL_REGISTERED_CALLBACK64,
+    CBA, PIMAGEHLP_CBA_EVENTW)
+from procmon_parser.symbol_resolver.win.win_types import PVOID, HANDLE, DWORD64, DWORD, ULONG, ULONG64, BOOL
+
+logger = logging.getLogger(__name__)
```

Comment:
Hmm, I'm used to use a named logger at the top of the file (as given in the [logging cookbook](https://docs.python.org/3/howto/logging.html#logging-advanced-tutorial) in the official doc).

If the logger is moved to the class (or if using `logging`), I gather that it would not be possible anymore to act upon the logger externally. For ex.  from a program that use the `procmon-parser`, and set the logging to DEBUG: in this case - with a module level logger - you can easily change the level or disable the logger in the `symbol_resolver` module.

Not sure you can do that if the logger is in the class (or using `logging`). 

I'd be happy to change that, as long as users can act upon it easily (e.g. `logging.getLogger('procmon_parser').setLevel(logging.CRITICAL)` and not resort to check all loggers (`logging.Logger.manager.loggerDict`).

Reply:
hmmm makes sense you can't argue with the official docs :)